msyncd/msyncd-app.pro

```diff
@@ -59,13 +59,19 @@ target.path = /usr/bin/
 loglevel.files = bin/set_sync_log_level
 loglevel.path = /etc/buteo/
 service.files = bin/msyncd.service
+upstart.files = bin/msyncd.conf
 service.path = /usr/lib/systemd/user/
+upstart.path = /usr/share/upstart/sessions/
```

Comment:
Upstart lines should probably be next to each other like all the other targets were.

Also if added here, we need to %exclude on rpm. Could maybe have CONFIG option to include this, or even move together with separate debian packaging?